topics/shell/excercise/answers/generate_password.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+generate_password() {
+  local length=$1
```

Comment:
The CI job is failing:
![image](https://github.com/tungbq/devops-basic/assets/85242618/7ad56b26-5115-4b0b-8bf4-95a5ee73af2a)

Can we make the default value of `length` to be 8 if there is no input so it can be run on the CI job as well?